optuna/samplers/_tpe/sampler.py

```diff
@@ -85,6 +85,13 @@ class TPESampler(BaseSampler):
       Problems <https://doi.org/10.1145/3377930.3389817>`__
     - `Multiobjective Tree-Structured Parzen Estimator <https://doi.org/10.1613/jair.1.13188>`__
 
+    Please also check our our articles:
+
+    - `Significant Speed Up of Multi-Objective TPESampler in Optuna v4.0.0
+      <https://medium.com/optuna/significant-speed-up-of-multi-objective-tpesampler-in-optuna-v4-0-0-2bacdcd1d99b>`__
+    - `Multivariate TPE Makes Optuna Even More Powerful
```

Comment:
Although this is optional, "This sampler is based on *independent sampling*" can be updated to be more precise since it is not always true now that the multivariate option exists (and I think setting `multivariate=True` is beneficial for users in most cases).
https://github.com/optuna/optuna/pull/5638/files#diff-2cc3a4088f95a10e806b68655d375f6f7f5fe2de161a0182c16168e378be8065R65

Reply:
You are right, the comment is obsolete.
I addressed your comment!